modules/rosapps/applications/sysutils/systeminfo/lang/de-DE.rc

```diff
@@ -32,18 +32,18 @@ Beispiele:\n\
   SYSTEMINFO /S system /FO CSV /NH\n"
 
 IDS_HOST_NAME,               "Domäne"
-IDS_OS_NAME,                 "OS Name"
-IDS_OS_VERSION,              "OS Version: Build %u"
+IDS_OS_NAME,                 "Betriebssystemsname"
+IDS_OS_VERSION,              "Betriebssystemsversion: Build %u"
 IDS_BUILD,                   "Build"
-IDS_OS_BUILD_TYPE,           "OS Build Typ"
+IDS_OS_BUILD_TYPE,           "Betriebssystems-Buildtyp"
 IDS_REG_OWNER,               "Registrierter Benutzer"
 IDS_REG_ORG,                 "Registrierte Organisation"
-IDS_PRODUCT_ID,              "Produkt ID"
+IDS_PRODUCT_ID,              "Produkt-ID"
 IDS_INST_DATE,               "Installationsdatum"
-IDS_UP_TIME,                 "System Up Time"
-IDS_UP_TIME_FORMAT,          "%u Days, %u Hours, %u Minutes, %u Seconds"
-IDS_SYS_MANUFACTURER,        "System Manufacturer"
-IDS_SYS_MODEL,               "System Model"
+IDS_UP_TIME,                 "Systemsuptime"
```

Comment:
No. I have no better translation because Uptime is also used in German but that sounds shitty

Reply:
I changed it to Betriebszeit in my latest commit (will push soon)